Use our devel/cargo MODULE to build instead of a custom target that needs
network access to fetch crates.

Index: src/meson.build
--- src/meson.build.orig
+++ src/meson.build
@@ -23,24 +23,3 @@ run_command(
   meson.project_source_root() / 'src' / 'config.rs',
   check: true,
 )
-
-cargo_build = custom_target(
-  'cargo-build',
-  build_by_default: true,
-  build_always_stale: true,
-  output: meson.project_name(),
-  console: true,
-  install: true,
-  install_dir: bindir,
-  depends: resources,
-  env: cargo_env,
-  command: [
-    cargo,
-    'build',
-    cargo_build_options,
-    '&&',
-    'cp',
-    cargo_target / rust_target / meson.project_name(),
-    '@OUTPUT@',
-  ],
-)
